lib/jsonb_accessor/query_builder.rb

```diff
@@ -81,7 +81,9 @@ module QueryBuilder
         JsonbAccessor::QueryHelper.validate_column_name!(all, column_name)
         JsonbAccessor::QueryHelper.validate_field_name!(all, column_name, field_name)
         JsonbAccessor::QueryHelper.validate_direction!(direction)
-        order("(#{table_name}.#{column_name} -> '#{field_name}') #{direction}")
+        sql = "(#{table_name}.#{column_name} -> '#{field_name}') #{direction}"
+        arel_sql_supported = Gem.ruby_version.release >= Gem::Version.new('2.2.5')
```

Comment:
This change shouldn't be necessary to get tests to pass. Tests are failing because the Travis script is trying to fetch rubygems 3.0, which has a minimum requirement of ruby 2.3.

Instead, you can revert https://github.com/madeintandem/jsonb_accessor/pull/107/commits/2daf3114fad102cdb4bd6ac5937f43aece60837f, and make the following tweak to `.travis.yml`:

```yml
before_install:
  - gem i rubygems-update -v '<3' && update_rubygems
  # ...
```

This restricts the rubygems update to < v3, which should allow tests for ruby 2.2.5 to run as well.

See https://github.com/rubygems/rubygems/issues/2534

Reply:
Another approach is to pull in the changes from https://github.com/madeintandem/jsonb_accessor/pull/94, since the library looks like it may be moving towards testing against newer ruby versions.